sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveScriptTransformationSuite.scala

```diff
@@ -546,14 +547,39 @@ class HiveScriptTransformationSuite extends BaseScriptTransformationSuite with T
             AttributeReference("a", DayTimeIntervalType())(),
             AttributeReference("b", DayTimeIntervalType())(),
             AttributeReference("c", DayTimeIntervalType())(),
-            // TODO(SPARK-35772): Check all year-month interval types in HiveInspectors tests
             AttributeReference("d", YearMonthIntervalType())()),
```

Comment:
Since you check `YearMonthIntervalType()` in separate test, please, remove this (and don't forget to fix test's title).

Reply:
> Since you check `YearMonthIntervalType()` in separate test, please, remove this (and don't forget to fix test's title).

Yea, I will update DayTimeIntervalType's test just in this UT